### PR TITLE
fix: convert story point to signed integer

### LINKED ISSUE #2380: [Bug][jira] 18446744073709551615 is greater than maximum value for Int8
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-devlake/issues?q=is%3Aissue) and found no similar issues.

### What happened

When I running jira task to collect jira data, it failed with error `18446744073709551615 is greater than maximum value for Int8`.  

### What you expected to happen

It should not cause error.

### How to reproduce

Not sure, I think it is related to my jira source data.

### Anything else

Some logs:
```
INSERT INTO \"_tool_jira_issues\" (\"connection_id\",\"issue_id\",\"project_id\",\"self\",\"icon_url\",\"issue_key\",\"summary\",\"type\",\"epic_key\",\"status_name\",\"status_key\",\"story_point\",\"original_estimate_minutes\",\"aggregate_estimate_minutes\",\"remaining_estimate_minutes\",\"creator_account_id\",\"creator_account_type\",\"creator_display_name\",\"assignee_account_id\",\"assignee_account_type\",\"assignee_display_name\",\"priority_id\",\"priority_name\",\"parent_id\",\"parent_key\",\"sprint_id\",\"sprint_name\",\"resolution_date\",\"created\",\"updated\",\"spent_minutes\",\"lead_time_minutes\",\"std_story_point\",\"std_type\",\"std_status\",\"all_fields\",\"changelog_updated\",\"remotelink_updated\",\"worklog_updated\",\"created_at\",\"updated_at\",\"_raw_data_params\",\"_raw_data_table\",\"_raw_data_id\",\"_raw_data_remark\") 
(1,20351,10003,'https://merico.atlassian.net/rest/agile/1.0/issue/20351','https://merico.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10315?size=medium','EE-9283','全站帮助系统规范视觉化','Story','','打开','new',-1.000000,0,0,0,'623ae914a9575800695a3b6c','','张彩霞','6062b11ed115550069901e2d','','朴文峰',3,'Medium',0,'',0,'',NULL,'2022-06-28 15:06:33.484','2022-06-28 15:20:11.206',0,0,18446744073709551615,'REQUIREMENT','TODO',NULL,'2022-06-29 02:57:09.4',NULL,NULL,'2022-06-29 02:57:55.319','2022-06-29 02:57:55.319','{\"ConnectionId\":1,\"BoardId\":68}','_raw_jira_api_issues',9686,'')
(1,20352,10003,'https://merico.atlassian.net/rest/agile/1.0/issue/20352','https://merico.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10315?size=medium','EE-9284','将指标说明详细文档（产品手册）内嵌到ee中','Story','','打开','new',0.000000,0,0,0,'623ae914a9575800695a3b6c','','张彩霞','60506cfdf180c300673626a9','','Xiaoxi Gu',3,'Medium',0,'',0,'',NULL,'2022-06-28 15:48:57.889','2022-06-28 16:09:05.277',0,0,0,'REQUIREMENT','TODO',NULL,'2022-06-29 02:57:09.4',NULL,NULL,'2022-06-29 02:57:55.319','2022-06-29 02:57:55.319','{\"ConnectionId\":1,\"BoardId\":68}','_raw_jira_api_issues',9687,'')
```

### Version

v0.11.0

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by mindlesscloud**
# Summary

fix #2380 [Bug][jira] 18446744073709551615 is greater than maximum value for Int8

The type of column `std_story_point` was unsigned integer, the value returned from API was `-1`, converting `-1` to unsigned interger we got `18446744073709551615`. To fix this, we alter the type of column `std_story_point` to a signed integer.
### Does this close any open issues?
close #2380 

### Screenshots
Include any relevant screenshots here.

### Other Information
Any other information that is important to this PR.

### CODE REVIEW

**Comment by likyh** [comment]
Maybe you can target to add some e2e tests.

**Comment by mindlesscloud** [comment]
> Maybe you can target to add some e2e tests.

Good idea

File: plugins/jira/tasks/issue_extractor.go [hunk 1]
```diff
@@ -100,10 +101,14 @@ func ExtractIssues(taskCtx core.SubTaskContext) error {
 				issue.LeadTimeMinutes = uint(issue.ResolutionDate.Unix()-issue.Created.Unix()) / 60
 			}
 			if data.Options.TransformationRules.StoryPointField != "" {
-				strStoryPoint := apiIssue.Fields.AllFields[data.Options.TransformationRules.StoryPointField].(string)
-				issue.StoryPoint, _ = strconv.ParseFloat(strStoryPoint, 32)
+				fmt.Println(apiIssue.Fields.AllFields[data.Options.TransformationRules.StoryPointField])
```
Comment (klesh) [inline plugins/jira/tasks/issue_extractor.go:104]: To be removed

File: plugins/jira/tasks/issue_extractor.go [hunk 2]
```diff
@@ -100,10 +101,14 @@ func ExtractIssues(taskCtx core.SubTaskContext) error {
...
+				fmt.Println(apiIssue.Fields.AllFields)
```
Comment (klesh) [inline plugins/jira/tasks/issue_extractor.go:105]: Same as above

**Review by mappjzc — APPROVED** [review]

**Pull request merged**